oligo_designer_toolsuite/IO/_database.py

```diff
@@ -112,196 +132,162 @@ def read_reference_DB(self, file_reference_DB):
-                        probe_attributes["strand"],
-                    )
-                    # if we computed additional features we write also those
-                    if len(columns) > 8:
-                        for column in columns[8:]:
-                            output += "\t{}".format(probe_attributes[column])
-                        # \n at the end f the string
-                    output += "\n"
-                    handle_probe.write(output)
+        if dir_oligos_DB is None:
+            self.dir_oligos_DB = self.dir_annotation
+        else:
+            self.dir_oligos_DB = os.path.join(self.dir_output, dir_oligos_DB)
+            Path(self.dir_oligos_DB).mkdir(parents=True, exist_ok=True)
+
+        if format == "tsv":
+            self.file_oligos_DB_tsv = os.path.join(
+                self.dir_oligos_DB,
+                self.file_name_oligos_DB_tsv,
+            )
+            data_parser.write_oligos_DB_tsv(self.oligos_DB, self.file_oligos_DB_tsv)
+            return self.file_name_oligos_DB_tsv
+        elif format == "gtf":
+            self.file_oligos_DB_gtf = os.path.join(
+                self.dir_oligos_DB,
+                self.file_name_oligos_DB_gtf,
+            )
+            self.file_oligos_DB_fasta = os.path.join(
+                self.dir_oligos_DB,
+                self.file_name_oligos_DB_fasta,
+            )
+            data_parser.write_oligos_DB_gtf(
+                self.oligos_DB, self.file_oligos_DB_gtf, self.file_oligos_DB_fasta
+            )
+            return self.file_name_oligos_DB_gtf, self.file_name_oligos_DB_fasta
+        else:
+            raise ValueError(f"{format} not recognized as a format!")
 
     def create_reference_DB(
         self,
-        genome=False,
-        gene_transcript=True,
-        gene_CDS=False,
-        dir_output="./output/annotation",
+        region="gene_transcript",
         block_size=None,
+        dir_reference_DB=None,
     ):
-        """Creates a fasta file for each of the region selected (genome, gene_transcript, gene_CDS) which will be used for alignements.
-        If not specified the exon juctions size is set to <probe_length_max> + 5.
-
-        :param genome: create the reference file for the whole genome, defaults to False
-        :type genome: bool, optional
-        :param gene_transcript: create the reference file for the gene transcript, defaults to True
-        :type gene_transcript: bool, optional
-        :param gene_CDS: create the reference file for the coding region, defaults to False
-        :type gene_CDS: bool, optional
-        :param dir_output: folder where the fasta file will be saved, defaults to './output/annotation'
-        :type dir_output: str, optional
-        :param block_size: size of the exon junctions, defaults to None
+        """Creates a fasta file for each of the region selected (genome, gene_transcript, gene_CDS) which will be used for alignements, default is "gene_transcript".
```

Comment:
...specified the exon **junctions** size is set to...
...**subdirectory** of dir_out where...